specification/amqp_0.9.1/pom.xml

```diff
@@ -0,0 +1,124 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.kaazing</groupId>
+        <artifactId>k3po.parent</artifactId>
+        <version>develop-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>specification.amqp.0.9.1</artifactId>
+
+    <name>AMQP Protocol Specification</name>
+    <url>https://github.com/k3po/specification.amqp.0.9.1</url>
+    <description>Specification for AMQP 0.9.1 protocol</description>
+    <packaging>jar</packaging>
+
+    <scm>
+        <url>https://github.com/k3po/specification.amqp.0.9.1.git</url>
+        <connection>scm:git:https://github.com/k3po/specification.amqp.0.9.1.git</connection>
+    </scm>
+
+    <properties>
```

Comment:
You don't need this <properties> element. The pom file in the specifications/ws project is a good reference to see what you need in this pom file.